pimd/pim_igmp.c

```diff
@@ -1031,8 +1031,13 @@ static int pim_igmp_read(struct thread *t)
 {
 	uint8_t buf[10000];
 	struct gm_sock *igmp = (struct gm_sock *)THREAD_ARG(t);
```

Comment:
No point updating this code, it's not used for IPv6.

Reply:
Ack